### PR TITLE
A Cython bundle

### PR DESCRIPTION
**Pull request by shalabhc**

### CODE REVIEW

File: bundles/cython/cython_mode.moon [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- Copyright 2017 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+{
+  lexer: bundle_load('cython_lexer')
+  structure: (editor) =>
+    [l for l in *editor.buffer.lines when l\match('^%s*class%s') or l\match('^%s*def%s') or l\match("^%s*cdef%s") or l\match("^s*cpdef%s")]
```
Comment (nilnor) [inline bundles/cython/cython_mode.moon]: Last pattern looks wrong.

File: bundles/cython/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Cython bundle
+
+## What is it?
+
+This is a [Cython](http://cython.org) bundle for the Howl editor, providing
+Cython specific lexing and a Cython mode. It is automatically enabled for
+Cython extensions such as `pyx` and `pyi`.
```
Comment (nilnor) [inline bundles/cython/README.md]: Mismatch with actual extensions in mode reg.

**Comment by nilnor** [comment]
Looks good apart from the two minor comments!

**Comment by shalabhc** [comment]
Ah - good catches. Fixed!

**Comment by nilnor** [comment]
:+1: Don't forget the changelog

**Comment by shalabhc** [comment]
Doh! Done now.

**Pull request merged**